src/main/java/org/folio/rest/impl/ServicePointAPI.java

```diff
@@ -34,6 +35,8 @@
   public static final String SERVICE_POINT_TABLE = "service_point";
   public static final String LOCATION_PREFIX = "/service-points/";
   public static final String ID_FIELD = "'id'";
+  public static final String SERVICE_POINT_CREATE_ERR_MSG_WITHOUT_HOLD_EXPIRY = "Hold shelf expiry period must be specified when service point can be used for pickup.";
+  public static final String SERVICE_POINT_CREATE_ERR_MSG_WITHOUT_BEING_PICKUP_LOC= "Hold shelf expiry period cannot be specified when service point cannot be used for pickup";
```

Comment:
Should these be translatable? I see we use `Messages` from RMB in other APIs. I also see we don't use it in some APIs, so it isn't clear what our i18n policy is here. Also, there is a space missing here: "SERVICE...LOC="

Reply:
We do not currently support translations in the backend. There was some early experimental support in RAML Module Builder, however it is rarely used now.

There are a few examples of validation messages with `code` properties that the UI could use for translations. However, I'm deferring specifically introducing those until we have a decision that this is the agreed approach, see https://issues.folio.org/browse/FOLIO-1716